src/navigation/tabs/settings/general/screens/Theme.tsx

```diff
@@ -14,11 +19,76 @@ import {AppActions} from '../../../../../store/app';
 import {logSegmentEvent} from '../../../../../store/app/app.effects';
 import {LogActions} from '../../../../../store/log';
 import {Settings, SettingsContainer} from '../../SettingsRoot';
+import {HeaderTitle} from '../../../../../components/styled/Text';
+import {useNavigation} from '@react-navigation/native';
+import {useAppDispatch, useAppSelector} from '../../../../../utils/hooks';
+import {Network} from '../../../../../constants';
+import {
+  networkChanged,
+  showBottomNotificationModal,
+} from '../../../../../store/app/app.actions';
+import {sleep} from '../../../../../utils/helper-methods';
+import RNRestart from 'react-native-restart';
 
 const ThemeSettings: React.FC = () => {
   const {t} = useTranslation();
-  const dispatch = useDispatch();
-  const currentTheme = useSelector(({APP}: RootState) => APP.colorScheme);
+  const dispatch = useAppDispatch();
+  const currentTheme = useAppSelector(({APP}: RootState) => APP.colorScheme);
+  const navigation = useNavigation();
+  const [clickCount, setClickCount] = useState(0);
+  const network = useAppSelector(({APP}) => APP.network);
+
+  const onPressTitle = () => {
+    const _clickCount = clickCount + 1;
+    setClickCount(_clickCount);
+    if (_clickCount >= 10) {
+      const changeNetwork =
+        network === Network.mainnet ? Network.testnet : Network.mainnet;
+
+      dispatch(
+        showBottomNotificationModal({
+          type: 'info',
+          title: `${t('Test Mode ')}${
```

Comment:
Don't concatenate partial sentence translations, use interpolation instead

tsx
`t(Test Mode )t(Enabled)` => `t((TestModeArg, {status: t(Enabled)})`

translation.json
`"Test Mode ": "Test Mode "` =>  `"TestModeArg": "Test Mode {{status}}"`

This way in case the order is different in another language (eg. JP, AR, etc) it translates better